src/index.css

```diff
@@ -60,11 +64,3 @@ button {
   background-color: transparent;
```

Comment:
background-color in design

![image](https://github.com/DavidTereshchenko/fe-react-2024/assets/108006836/f426113b-e0dd-4cfd-822d-443d5d5a5f9a)